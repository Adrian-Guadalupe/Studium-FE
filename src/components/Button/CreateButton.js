import React from 'react';
import styled from 'styled-components';
import PropTypes from 'prop-types';
import { Button } from 'antd';

export const CreateButton = ({
    text, type = 'primary', size = 'default', icon, shape, loading, block, color, ...props
}) => {
    
}