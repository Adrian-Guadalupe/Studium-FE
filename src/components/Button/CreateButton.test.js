import React from 'react';
import renderer from 'react-test-renderer';

import {CreateButton} from './CreateButton.js';

test('Create Button works', () => {
    const tree = renderer.create(<CreateButton />).toJSON();
    expect(tree).toMatchSnapshot();
})
