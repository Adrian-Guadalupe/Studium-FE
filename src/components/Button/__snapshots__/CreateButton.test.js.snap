// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Button works 1`] = `
.c0.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  position: relative;
  background: #4CB69F;
  border-radius: 100%;
  height: 69px;
  width: 69px;
  border: 7px solid white;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0.c0 > Svg {
  height: 33px;
  width: 33px;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  position: absolute;
}

<button
  className="ant-btn c0"
  onClick={[Function]}
  type="button"
>
  <svg>
    Union.svg
  </svg>
</button>
`;
